.github/workflows/plugin_tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Plugin TasksTimetable Tests
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
On travis we had the tests setup in a way, that they were triggered once a week. Don't think we need to run them daily...

Reply:
@removed